DigitalLearningSolutions.Data.Tests/Services/RegistrationServiceTests.cs

```diff
@@ -1126,6 +1164,62 @@ public void CreateDelegateAccountForExistingUser_with_unapproved_IP_registers_de
             approved.Should().BeFalse();
         }
 
+        [Test]
+        public void CreateDelegateAccountForExistingUser_adds_new_delegate_to_groups()
+        {
+            // Given
+            const int userId = 2;
+            const int delegateId = 2;
+            var model = RegistrationModelTestHelper.GetDefaultInternalDelegateRegistrationModel();
+            var delegateEntity = UserTestHelper.GetDefaultDelegateEntity(
+                7,

```

Comment:
I am confused by this mock. Why are you specifying that `userDataService.GetDelegateById(2)` returns a delegate entity with delegate id = 7?